cli/cmd/root.go

```diff
@@ -67,6 +69,12 @@ func NewCmdRoot() *cobra.Command {
 			// PersistentPreRunE runs after argument parsing, so errors during parsing will result in printing the help
 			cmd.SilenceUsage = true
 			var err error
+
+			invocationUUID, err = uuid.NewRandom()
+			if err != nil {
+				return fmt.Errorf("failed to generate invocation uuid: %w", err)
```

Comment:
If someone is using this to monitor there syncs then I think it is better to error out rather than run something that is not able to be properly monitored